clusters/build-clusters/02_cluster/README.md

```diff
@@ -0,0 +1,17 @@
+# 01-Cluster
+
+[02-Cluster](https://console-openshift-console.apps.build02.gcp.ci.openshift.org) is an OpenShift-cluster managed by DPTP-team. It is one of the clusters for running Prow job pods.
+
+## Installation
+
+The gcp project `openshift-ci-build-farm` for installation of this cluster: [gcp console](https://console.cloud.google.com/home/dashboard?project=openshift-ci-build-farm). The project is created by [DPP team](hhttps://issues.redhat.com/browse/DPP-4926).
+
+We created the public zone (base domain for installer) on the project: `ci.devcluster.openshift.com` and the
+service account `ocp-cluster-installer` whose key is stored as an attachment of BitWarden item `build_farm_02_cluster`.
+
+Since this project is dedicated for build farm. We granted the `Owner` role to this account.
+
+TODO:
+
+* Choose version of installer.
```

Comment:
@stevekuznetsov 
What about https://openshift-release.svc.ci.openshift.org/releasestream/4-stable/release/4.4.4
The latest release version?

Reply:
Sure -- let's be in the same channel

Reply:
Build01 uses the `candidate-4.4` channel.
I will do it for `build02` after installation. By default, it should be `stable-4.4`.

Reply:
<img width="943" alt="Screen Shot 2020-05-21 at 4 34 59 PM" src="https://user-images.githubusercontent.com/4013349/82603688-1eea9d00-9b81-11ea-956a-46a5128eadec.png">

Changed manually on the UI to the `candidate-4.4` channel.

Reply:
BTW, an update is available. Which is accepted 3 hours ago!!!